CONTRIBUTING.md

```
@@ -0,0 +1,79 @@
+- [ ] Your branch name follows our naming conventions
+- [ ] Your code passes CI and lint checks
+- [ ] You’ve added tests or explained why not
+- [ ] Your logic is documented where needed
+- [ ] All review comments have been addressed or responded to
+
+---
+
+## 🤖 Code Review
+
+Robotick uses **CodeRabbit** for AI-assisted reviews:
+- It checks for bugs, unsafe logic, style, and missing comments
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)